src/platforms/java/common/configuration/index.mdx

```diff
@@ -133,6 +133,17 @@ To set the server name that will be sent with each event, use the `servername` o
 servername=host1
 ```
 
+### Timeout (Advanced)
+
+A timeout is set to avoid blocking Sentry threads because establishing a connection is taking too long.
+
+It’s possible to manually set the timeouts length with `connectionTimeoutMillis` and `readTimeoutMillis`:
+
+```
+connectionTimeoutMillis=10000
+readTimeoutMillis=10000
+```
```

Comment:
```suggestion
```java
Sentry.init(options -> {
    options.setConnectionTimeoutMillis(10000);
    options.setReadTimeoutMillis(10000);
});
```
```

Reply:
Since these properties cannot be changed using external configuration I moved this section one level up and applied your suggested change.